flutter-studio/build.gradle.kts

```diff
@@ -52,15 +52,12 @@ intellij {
 
   // If 2023.3+ and IDEA (not AS), then "org.jetbrains.android:$androidVersion", otherwise "org.jetbrains.android",
   // see https://github.com/flutter/flutter-intellij/issues/7145
-  if(ide == "android-studio") {
+  if (ide == "android-studio") {
     pluginList.add("org.jetbrains.android")
   } else if (ide == "ideaIC") {
     pluginList.add("org.jetbrains.android:$androidVersion")
   }
 
-  if (ideVersion != "2023.2") {
-    pluginList.add(smaliPlugin)
```

Comment:
So is this plugin not needed anymore when running in Android Studio?

Reply:
This logic was used when building the Flutter plugin for the `2023.2` IDEA version of the product, at this point however we won't be pushing out a version of the Flutter plugin for `2023.2`: https://github.com/flutter/flutter-intellij/blob/07610c1301ce8792975d10274266abc97e4b2699/product-matrix.json

Reply:
This logic looks like we wanted the `smaliPlugin` for all versions that were **not** 2023.2, including newer versions. So does that mean we should be changing this from
```
if (ideVersion != "2023.2") {
  pluginList.add(smaliPlugin);
}
```
to
```
pluginList.add(smaliPlugin);
```